packages/stencil/src/components/datacom-list/datacom-li.css

```diff
@@ -15,14 +15,14 @@
   font-weight: 400;
 }
 
-li::marker {
-  font-size: 400;
-  line-height: 20px;
-  text-align: right;
-  position: absolute;
-  letter-spacing: -0.25px;
+.dc-li-heading {
+  font-weight: 700;
 }
 
-.dc-li-heading {
+.dc-li-marker::marker {
   font-weight: 700;
 }
+
+.dc-li-item.dc-li-marker {
```

Comment:
Do you need the .dc-li-item class in this selector for the margin to apply? If we can avoid being super specific with our selectors it would help with readability and maintainability.

Reply:
Yes, the margin is not applied to this particular place.

Reply:
Cool. Thanks for confirming that. Guess we'll have to live with it.